packages/server-web/src/lib/tracked-time-extractor.js

```diff
@@ -0,0 +1,20 @@
+module.exports = { extractTrackedTimes };
+
+function extractTrackedTimes(inputString) {
+  const regex = /#TRACK\((\{.*?\:.*?\})\)/gi;
+  const matches = inputString.match(regex);
+  let text = inputString;
+  let trackedTimes = [];
+  if (matches) {
+    trackedTimes = matches.map(match => {
+      text = text.replace(match, "");
+      // in theory, this should be provided as capture group by the regex engine.
+      // not sure why it doesn't work
```

Comment:
I think you want to use `while ((matches = regex.exec(string)) !== null) { ... }`

Reply:
There is a difference betweens `myString.match(regex)` and `regex.exec(myString)` I guess

Reply:
yes, that did the trick! thx